src/main/kotlin/io/appwrite/models/ContinentList.kt

```diff
@@ -8,7 +8,7 @@ import io.appwrite.extensions.jsonCast
  */
 data class ContinentList(
     /**
-     * Total number of continents documents that matched your query.
+     * Total number of continents rows that matched your query.
```

Comment:
the sentence feels... odd to read? not a blocker though.

Reply:
Unexpected, not sure where these ones come from, probably the `BaseList`. Checking